meter/openwb.go

```diff
@@ -24,6 +24,7 @@ func NewOpenWBFromConfig(other map[string]interface{}) (api.Meter, error) {
 		Topic       string
 		Timeout     time.Duration
 		Usage       string
+		Capacity    *float64 `mapstructure:"capacity"`
```

Comment:
For the later call to `NewConfigurable()` that requires it. 
But now that I think about it, it's indeed kind of useless on ameter that does not support batteries.
However, this could be enhanced later to allow Chargers to return their vehicle's Capacity, which is why I added the new interface to Vehicle.

Reply:
> For the later call to NewConfigurable() that requires it.

Just pass `nil`

Reply:
ok.